bootstrap_models.py

```diff
@@ -1,11 +1,21 @@
 import sys
 import os
 import subprocess
+from dotenv import load_dotenv
+from huggingface_hub import HfApi
 
 DEFAULT_EMBEDDINGS_MODEL ="sentence-transformers/all-MiniLM-L6-v2"
 DEFAULT_MODEL = "openlm-research/open_llama_3b"
 MODEL_DIR = "./models"
 
+def login_to_huggingface_hub():
+    load_dotenv()
+    hf_token = os.getenv("HUGGINGFACE_TOKEN")
+
+    # Login to Hugging Face Hub
+    hf_api = HfApi()
+    user = hf_api.whoami(hf_token)
```

Comment:
make this conditional to there being a huggingface token present in `.env`. If not, continue with unauthenticated session.